dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -294,6 +294,7 @@ public static void removeFixTask(Class<?> clazz){
         .add(Task201014UpdateColumnsValuesInIdentifierTable.class)
 		.add(Task210218MigrateUserProxyTable.class)
 		.add(Task210316UpdateLayoutIcons.class)
+		.add(Task210304RemoveOldMetadataFiles.class)
```

Comment:
This needs to be moved up before **Task210316UpdateLayoutIcons** b/c in the upgrade process we check that the taskId should be higher than the current db_version.

If this is not going to be merged for this release, we'll need to update this taskId to a greater Id (older date) otherwise will not run.

Reply:
done